src/main/java/athena/Athena.java

```diff
@@ -1,46 +1,46 @@
 package athena;
 
 import athena.commands.Command;
+import athena.exceptions.CommandException;
+
+import java.util.Scanner;
 
 public class Athena {
-    private static Ui ui;
+    private Ui ui;
+    private Parser parser;
+    private Storage storage;
+    private TaskList taskList;
 
     public Athena() {
         ui = new Ui();
+        parser = new Parser();
+        storage = new Storage("save.csv", ui);
```

Comment:
the correct name here according to the user guide is load.csv. This was my mistake, eventually we should allow the user to change what file they are loading

Reply:
This was taken from #55. Can you comment over there?

Reply:
Done